src/test/java/sync/google/GoogleKeywordDownloaderTest.java

```diff
@@ -34,6 +39,64 @@ public void setup() {
     }
 
     @Test
+    public void testDownloadTwoGoogleKeywordsBelongsToSameCampaignId(){
+        setupKeywordsInGoogle(
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Active").withText("shoes"),
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Paused").withText("shirts")
+        );
+
+        List<KenshooKeyword> kenshooKeywords = testedDownloader.download(
+                    selector().withCampaignIdInTarget(CAMPAIGN_ID).build()
+        );
+
+        assertThat(kenshooKeywords, hasSize(2));
+    }
+
+    @Test
+    public void testDownloadActiveGoogleKeyword(){
+        setupKeywordsInGoogle(
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Active").withText("shoes")
+        );
+
+        List<KenshooKeyword> kenshooKeywords = testedDownloader.download(
```

Comment:
Pretty good!
You can make it even easier to read if you create a method that is used like this:
```
List<KenshooKeyword> kenshooKeywords = downloadKeywords(using(selector().withCampaignIdInTarget(1000));
```
* downloadKeywords - invokes the tested downloader
* using - a method that does nothing. (just returns the same arg it was called with)
              I suggest this method to make the statement readable like an english statement.